docs/command/atlas-api-teams-getProjectTeam.txt

```diff
@@ -0,0 +1,104 @@
+.. _atlas-api-teams-getProjectTeam:
+
+==============================
+atlas api teams getProjectTeam
+==============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+`Public Preview: please provide feedback <https://feedback.mongodb.com/forums/930808-atlas-cli>`_: Returns one team to which the authenticated user has access in the project specified using its unique 24-hexadecimal digit identifier.
+
+Public Preview: The atlas api sub-command, automatically generated from the MongoDB Atlas Admin API, offers full coverage of the Admin API and is currently in Public Preview (please provide feedback at https://feedback.mongodb.com/forums/930808-atlas-cli).
```

Comment:
```suggestion
Public Preview: The ``atlas api`` sub-command, automatically generated from the MongoDB Atlas Admin API, offers full coverage of the Admin API and is currently in Public Preview (please provide feedback at https://feedback.mongodb.com/forums/930808-atlas-cli).
```
I think we should consider monospacing this, since it's referring specifically to a command.